### PR TITLE
Automated cherry pick of #4263: update enterprise trial url

### PR DESCRIPTION
**Pull request by bcreane**
Cherry pick of #4263 on release-v3.13.

#4263: update enterprise trial url

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Review by lmm — APPROVED** [review]

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
